libs/core/src/scss/base/_variables.scss

```diff
@@ -88,19 +88,12 @@ $border-radius-round: 999px !default;
 /* Breakpoints
 ****************************************************************************/
 $breakpoints: (
-  small: 321px,
-  medium: 721px,
-  large: 1025px,
-  xlarge: 60em,
-  xxlarge: 80em,
+  xsmall: 321px,
+  small: 632px,
```

Comment:
Beware of breaking change!

@jakobe @jkaltoft @mark-drastrup and me agree that since we are doing this as a breaking change, we can update these values to 
XS >= 320
SM >= 632
MD >= 768
LG >= 992 
XL >= 1440 

as suggested. But this means that everywhere the small breakpoint is used we need to update it to use xsmall internally, and also alert users that they should change this in the few places they use small breakpoint.